### DISCUSSION TITLE
What name should we give this browser?

### DISCUSSION THREAD

**Discussion by uazo** [uazo/bromite-buildtools#202] [Polls]
I had to make a selection because the polls only allow 8 answers, so... these are the ones I like.

see https://github.com/uazo/bromite-buildtools/issues/171

**Comment by leacha69** [comment]
I voted Cromite. That and a silver/black logo.

**Comment by mspacek** [comment]
@uazo sorry to drop this here, but it hit me middle of last night: a name that implies a cleaned up version of chrome, without all the invasive crap, might be appropriate. All of these have little to no hits on GitHub, and have at least a few domain names open:

* `cleant` .io .me
* `cleand` .org .app
* `chlorite` .org .app

* `kleant` .org .app
* `kleand` .org .app
* `klorite` .com .org .app

**Comment by leacha69** [comment]
Chlorite is 👍🏻👍🏻. It harkens to its Bromite roots and all of the work done.

**Comment by nihal697** [comment]
no chromite?

**Reply by dioxias** [reply]
I think we should def refine the poll to popular choices and their variants at some point

**Comment by idviscious** [comment]
How about Brite, or Brite Browser? 
Either of these would maintain some provenance to Br(om)ite, while moving on from that chemical/Chromium/Google branding. 
And, 'brite,' has many positive connotations.

**Reply by dikiaap** [reply]
Awesome name but too late? :(
